charts/posthog/templates/_snippet-initContainers-wait-for-migrations.tpl

```diff
@@ -8,7 +8,12 @@
     - >
         until (python manage.py migrate --check);
         do
-            echo "Waiting for database migrations to be completed"; sleep 1;
+            echo "Waiting for PostgreSQL database migrations to be completed"; sleep 1;
+        done
+
+        until (python manage.py migrate_clickhouse --check);
+        do
+            echo "Waiting for ClickHouse database migrations to be completed"; sleep 1;
```

Comment:
LGTM, should we add as well a loop for `python manage.py run_async_migrations --check` and `python manage.py sync_replicated_schema`?

Reply:
Maybe, but I'd rather get those in separately if that's ok? I don't feel super comfortable on what the functionality should be for those ones.